docs/instruction_for_testing.md

```diff
@@ -0,0 +1,82 @@
+# Running tests before release
+
+The purpose of this document is to detail the steps to take to properly run the CLI integration tests on each supported platform. Not all test flows have been integrated in our automated pipeline so it is important to understand how to run tests in a semi-automatically fashion.
+
+It is **important** to note that this guide is not strictly for running tests while prepping for a release. This guide should be reviewed and the testing process defined here should be run a few weeks before a release, as you incrementally add logic. That way, if bugs are uncovered, you have enough time to address them!
+
+## Deployment and Setup
+
+Before we can run our integration tests, we need to deployment the required environments and create the necessary artifacts.
+
+At a high level, you will need to:
+
+1. Deploy and configure machines
+1. Clone the CLI/SDK repository
+1. Install dependencies on those machines
+1. Build the `integrations_tests_runner`
+
+#### Deploy and configure machines
+
+You will need a machine to host your Conjur server (should be Appliance) and 4 different EC2 machines (Windows, RHEL 7, RHEL 8, macOS - if available) for packing the executable. If macOS is not available, your local macOS machine should suffice.
```

Comment:
TODO - add clone the repository on those machines. You will need the project to build the artifact used for testing

Reply:
Done